sunpy/visualization/animator/tests/test_basefuncanimator.py

```diff
@@ -63,3 +114,54 @@ def test_edges_to_centers_nd():
     expected[:, edges_axis] = np.arange(10.5, 19)
     output = base.edges_to_centers_nd(axis_range, edges_axis)
     assert np.array_equal(output, expected)
+
+
+class ArrayAnimatorTest(ArrayAnimator):
+    def __init__(self, data):
+        self.naxis = data.ndim
+        self.image_axes = [1]
+        self.slider_axes = [0]
+
+    def plot_start_image(self, ax):
+        pass
+
+    def update_plot(self, val, artist, slider):
+        pass
```

Comment:
Adding the super call in here will enable you to test the code in the base class.

```suggestion
        super().update_plot(val, artist, slider)
```